.travis.yml

```diff
@@ -372,6 +451,8 @@ stages:
     if: type = cron
   - name: Deployments
     if: type = push AND branch =~ /^master|[0-9]+-(dev|stable|beta)|1-dev|$/
+  - name: SDKs snippets tests
+    if : type =~ /(cron|pull_request)/ OR (type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/)
```

Comment:
I'm not sure about the condition here because it means that the jobs will be executed on `master` as well but the SDK branches are `*-dev`

Reply:
Right, I'll adapt that

Reply:
Done in the last commit